### PR TITLE
resourceQuery and oneOf examples

### PR DESCRIPTION
**Pull request by nojvek**
Adding some examples. 

Hopefully this saves some precious hours of somebody else's life.

### CODE REVIEW

File: content/configuration/module.md [hunk 1]
```diff
@@ -133,6 +133,22 @@ W> This option is __deprecated__ in favor of `Rule.use`.
...
+  oneOf: [
+    {
+      resourceQuery: /inline/, // foo.css?inline
+      use: 'url-loader',
+    },
+    {
+      resourceQuery: /external/, // foo.css?external
+      use: 'file-loader',
+    },
+  ],
```
Comment (skipjack) [inline content/configuration/module.md]: Can you drop the trailing `,` in the example? I know it's valid ES6 but we've tried to stay away from it in the docs for consistency (it also looks a bit cleaner without them).

File: content/configuration/module.md [hunk 2]
```diff
@@ -175,6 +191,13 @@ A [`Condition`](#condition) matched with the resource. See details in [`Rule` co
 
 A [`Condition`](#condition) matched with the resource query. The condition matches against a string that starts with a question mark (`"?exampleQuery"`). See details in [`Rule` conditions](#rule-conditions).
 
+```javascript
+{
+  test: /.css$/,
+  resourceQuery: /inline/, // foo.css?inline
+  use: 'url-loader',
```
Comment (skipjack) [inline content/configuration/module.md]: Same here re commas.

**Comment by nojvek** [comment]
Fixed comma. Its interesting how trailing commas are polarizing akin to tabs vs spaces.

**Comment by nojvek** [comment]
@skipjack ping.

**Comment by skipjack** [comment]
> Fixed comma. Its interesting how trailing commas are polarizing akin to tabs vs spaces.

@nojvek thanks for fixing, and yeah I know what you mean. Similar to semicolons vs no semicolons as well 😆 ! Everyone has a different preference it seems but my main concern is just that we keep it consistent, at some point we may enforce some styling via eslint on markdown code blocks again (though it has caused issues in the past).

**Pull request merged**